src/component/Skeleton.js

```diff
@@ -0,0 +1,18 @@
+import { makeDom } from "./util";
+
+const skeleton = () => {
+  const dom = makeDom(
+    `<li class="skeleton-li">
+      <a href="#">
+        <div class="item-card">
+          <div class="item-thumbnail skeleton"></div>
+          <div class="item-title skeleton"></div>
+          <div class="item-score skeleton"></div>
+        </div>
+      </a>
+    </li>`
+  );
+
+  return dom;
+};
```

Comment:
```suggestion
const skeleton = () => makeDom(
    `<li class="skeleton-li">
      <a href="#">
        <div class="item-card">
          <div class="item-thumbnail skeleton"></div>
          <div class="item-title skeleton"></div>
          <div class="item-score skeleton"></div>
        </div>
      </a>
    </li>`
  );
```

이렇게 바로 반환해도 좋지 않을까 싶어요!

그리고 template literal을 더 발전시켜서

```suggestion
const skeleton = () => html`
  <li class="skeleton-li">
    <a href="#">
      <div class="item-card">
        <div class="item-thumbnail skeleton"></div>
        <div class="item-title skeleton"></div>
        <div class="item-score skeleton"></div>
      </div>
    </a>
  </li>
  `;
```

이런식으로 사용할 수 있도록 만들어줘도 좋아보이네요 ㅎㅎ

Reply:
html`` 이런식으로 코드를 작성하려면 어떻게 해야하는지 잘 모르겠네요. 어떤식으로 작성해야 하는지 좀 더 알려주시면 좋을 것 같습니다.